modular_skyrat/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -0,0 +1,159 @@
+	color = "#00014d60"
+	taste_description = "raw energy"
+	glass_icon_state = null
+	pH = 5 //slightly acidic
+	glass_name = "glass of Maintenance Energy"
+	glass_desc = "Unleash the tide."
+	var/list/healed_roles = list("Assistant")
+	var/heal_amount = 0.25
+
+/datum/reagent/consumable/maint_energy/on_mob_add(mob/living/L, amount)
+	. = ..()
+	shake_camera(L, amount, max(amount/10, 1))
+
+/datum/reagent/consumable/maint_energy/on_mob_life(mob/living/carbon/M)
+	. = ..()
+	M.Jitter(20)
+	M.drowsyness = max(M.drowsyness - 0.5, 0)
+	if(M.mind)
+		if((M.mind.assigned_role in healed_roles) || (M.mind.special_role in healed_roles))
+			M.adjustBruteLoss(-heal_amount, 0)
+			M.adjustFireLoss(-heal_amount, 0)
+			M.adjustToxLoss(-heal_amount, 0)
+			M.adjustOxyLoss(-heal_amount, 0)
+
+/datum/reagent/consumable/maint_energy/zero_fusion
+	name = "Maintenance Energy Zero Fusion"
+	description = "All the energy you need, without any calories."
+	nutriment_factor = 0.05 * REAGENTS_METABOLISM //Hey, isn't this thing supposed to be calorie-free?
+	color = "#ffffff3b"
+	taste_description = "sugar-free energy"
+	glass_name = "glass of Maintenance Energy Zero Fusion"
+	glass_desc = "Unleash the singulo."
+
+/datum/reagent/consumable/maint_energy/zero_fusion/on_mob_life(mob/living/carbon/M)
+	. = ..()
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.age >= 30)
+			M.adjustBruteLoss(-(2 * heal_amount), 0)
+			M.adjustFireLoss(-(2 * heal_amount), 0)
+			M.adjustToxLoss(-(2 * heal_amount), 0)
+			M.adjustOxyLoss(-(2 * heal_amount), 0)
+
+/datum/reagent/consumable/maint_energy/tritium_flood
+	name = "Maintenance Energy Tritium Flood"
+	description = "The flavor of atmosia, inside your mouth."
+	nutriment_factor = 0.65 * REAGENTS_METABOLISM //Has a lot of flavor.
+	pH = 4 //Acidic lemon flavor.
+	color = "#00ff3360"
+	taste_description = "radioactive lemon"
+	glass_name = "glass of Maintenance Energy Tritium Flood"
+	glass_desc = "Unleash the tesla."
+	healed_roles = list("Station Engineer", "Atmospheric Technician")
+
+/datum/reagent/consumable/maint_energy/plasma_fire
+	name = "Maintenance Energy Plasma Fire"
+	description = "The flavor of a hellburn, inside your mouth."
+	nutriment_factor = -0.1 * REAGENTS_METABOLISM //Burns your calories away.
+	color = "#ffd00073"
+	taste_description = "burning mango"
+	glass_name = "glass of Maintenance Energy Plasma Fire"
+	glass_desc = "Unleash the supermatter."
+	healed_roles = list("Station Engineer", "Atmospheric Technician")
+
+/datum/reagent/consumable/maint_energy/raid
+	name = "Maintenance Energy Raid"
+	description = "Energy drink seller, i am going into battle, and i require your strongest energy drinks."
+	nutriment_factor = 0.4 * REAGENTS_METABOLISM
+	color = "#ff002b96"
+	taste_description = "cherries"
+	glass_name = "glass of Maintenance Energy Raid"
+	glass_desc = "Unleash the Death Squad."
+	healed_roles = list("Captain", "Head of Security", "Head of Personnel", "Blueshield",\
```

Comment:
Dunno why you have `\` here, list will take everything in the brackets, even if closing bracket is in another ~~castle~~ line.
~~copied from somewhere?~~

Reply:
there really is no reason besides it's a practice that stuck to my head
i know that lists can be broken up into multiple lines but it's just because i'm an idiot